internal/tasks/tasks.go

```diff
@@ -931,6 +951,78 @@ func taskDisableTunnel() string {
 	return "{\"status\": \"ok\"}"
 }
 
+func taskStartShell(args taskStartShellArgs) string {
+	fmt.Println("Executing taskStartShell")
+	wsPath := utils.GetPath(utils.WsPath)
+
+	// kill the process if its running
+	utils.Exec(wsPath, "killall", []string{"sshx"})
+
+	cmd := exec.Command("/usr/local/bin/sshx", "--quiet", "--shell", "bash")
```

Comment:
Where are we installing sshx?

Reply:
We are not doing this in here, as generally the necessary dependency packages are installed using one of the setups (depending on the platform):

- https://github.com/edgebox-iot/multipass-setup (We should update this repo to include the installation of sshx on the setup script)
- https://github.com/edgebox-iot/ua-netinst-config (The same update is needed here)
- https://github.com/edgebox-iot/image-builder (this one is a bit out of date as of now so I guess we can skip adding it here, and we should consider that repo for archival).

I'll be creating PR's to add the sshx installation to the setup scripts.